starkinfra/brcodepreview/__brcodepreview.py

```diff
@@ -0,0 +1,70 @@
+from ..utils import rest
+from starkcore.utils.resource import Resource
+
+
+class BrcodePreview(Resource):
+    """# BrcodePreview object
+    A BrcodePreview is used to get information from a BR Code you received to check the informations before paying it.
+    ## Parameters (required):
+    - id [string]: Main identification of the payment. This should be the BR Code for Pix payments. ex: "34191.09008 63571.277308 71444.640008 5 81960000000062", "00020126580014br.gov.bcb.pix0136a629532e-7693-4846-852d-1bbff817b5a8520400005303986540510.005802BR5908T'Challa6009Sao Paulo62090505123456304B14A"
```

Comment:
BR Code (aka copia e cola) from a Pix payment. This is also de information directly encoded in a QR Code.

Reply:
Me parece que tem um exemplo de boleto aí a direita também, tem que arrancar

Reply:
Check